Rakefile

```diff
@@ -3,3 +3,25 @@
 require 'bundler/setup'
 
 require 'bundler/gem_tasks'
+require 'debug'
+
+desc 'Prepare a new version for release, version can be major, minor, patch or x.y.z (as per gem-release gem)'
+task :prepare_release, %i[version] do |_, args|
+  bump_version = args.fetch(:version)
+
+  current_branch = `git branch --show-current`.chomp
+  raise 'could not get current branch' if current_branch.empty?
+
+  sh 'git', 'checkout', '-b', 'new-release' if current_branch == 'main'
+
+  sh 'gem', 'bump', '-v', bump_version, '--no-commit'
+
+  version = `ruby -rrubygems -e 'puts Gem::Specification::load("dfe-analytics.gemspec").version'`.chomp
```

Comment:
How about

```suggestion
  version = `bundle exec ruby -e 'puts DfE::Analytics::VERSION'`.chomp
```

Reply:
Ah—see below!

Reply:
Thought about that, but liked the agnosticism of using `Gem::Specification`. Happy either way though, happy to switch this.

Reply:
Changed.